production_services/settings.py

```diff
@@ -73,36 +77,46 @@
     },
 ]
 
-WSGI_APPLICATION = 'ensembl_production_api.wsgi.application'
-
+WSGI_APPLICATION = 'production_services.wsgi.application'
 
 # Database
 # https://docs.djangoproject.com/en/1.11/ref/settings/#databases
 ensembl_version = os.getenv('ENS_RELEASE', '97')
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': BASE_DIR + '/db.sqlite3',
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': os.getenv('USER_DB_DATABASE', 'ensembl_production_services'),
+        'USER': os.getenv('USER_DB_USER', 'root'),
+        'PASSWORD': os.getenv('USER_DB_PASSWORD', ''),
+        'HOST': os.getenv('USER_DB_HOST', '127.0.0.1'),
+        'PORT': os.getenv('USER_DB_PORT', '3306'),
+        'OPTIONS': {
+           "init_command": "SET default_storage_engine=MYISAM",
+        }
     },
     'production': {
-        'ENGINE': os.getenv('PROD_ENGINE', 'django.db.backends.mysql'),
-        'NAME': 'ensembl_production_' + ensembl_version,
-        'USER': os.getenv('PROD_USER', 'root'),
-        'PASSWORD': os.getenv('PROD_PASSWORD', ''),
-        'HOST': os.getenv('PROD_HOST', '127.0.0.1'),
-        'PORT': os.getenv('PROD_PORT', '3306'),
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': os.getenv('PROD_DB_DATABASE', 'ensembl_production_97'),
```

Comment:
Why did you remove `ensembl_version` here? 
If you don't want to use it I guess you will need to cleanup the line above.